changelog

```diff
@@ -1,3 +1,280 @@
+    * Improved const correctness of various parameters and class member functions. (@CasparKielwein)
+    * File I/O now uses streams. (@tibetiroka)
+    * Made wider use of the PlayerInfo::AddPlayerSubstitutions method. (@warp-core)
+    * Text input fields in the conversation panel now support cut, copy, and paste, dialogs clipboard functionality now also supports cut. (@TomGoodIdea)
+    * Cropped the Astral Cetacean sprites, removing some empty pixels. (@Saugia)
+    * The "--matches" command line argument can now correctly handle paths traversing wormholes. (@warp-core)
+    * Fixed some style inconsistencies in planet definitions. (@warp-core)
+    * Improved the consistency of some header file names and #include statements. (@TomGoodIdea)
+    * Prevented ConditionStore objects from being copied. (@warp-core)
+    * Made ConditionEntry a top level class and simplified condition providers. (@petervdmeer)
+    * All ConditionSet objects now store a pointer to the player's ConditionsStore object. (@Amazinite, @petervdmeer)
+    * Reordered some members of the MapPanel class for consistency. (@TomGoodIdea)
+    * Added SSE2 code paths to the Point class. (@tibetiroka)
+    * Removed some incorrect comments from GameWindow.cpp. (@TomGoodIdea)
+    * The PlayerInfo and ConditionEntry classes can no longer be copied or moved. (@petervdmeer)
+    * Replaced some uses of std::string::compare with std::string::starts_with and std::string::ends_with. (@warp-core)
+    * Standardized use of "key" and "hasValue" variables in "Load" methods. (@Amazinite)
+    * Limited the minimum starfield zoom to 0.15 and added an option to not zoom the starfield with the view zoom. (@Amazinite, @TomGoodIdea)
+  * CI/CD and development environment:
+    * Marked the content style checker script as an executable file. (@SomeTroglodyte)
+    * Updated and improved the contributing file. (@roadrunner56, @Amazinite)
+    * Updated the name of the config file for editorconfig-checker. (@tibetiroka)
+    * Removed support for building with scons. (@petervdmeer)
+    * Mentioned using alternative linkers in the build instructions. (@Koranir)
+    * Fixed and re-added Windows x86 CD jobs. (@warp-core)
+    * Added an editorconfig entry for CMakeLists.txt files. (@TomGoodIdea)
+    * Added a CI check for commonly erroneously repeated words. (@Koranir)
+    * Fixed issues when trying to compile with cmake 4.0. (@warp-core, @TomGoodIdea)
+    * CI build jobs will now also be run on changes to vcpkg triplets in the "overlays" path. (@petervdmeer)
+    * Updated cmake version compatibility from 3.16-3.29 to 3.19-4.0. (@TomGoodIdea)
+    * The flagship autocondition integration test no longer takes place where it can be affected by random ion storms. (@Amazinite)
+    * Cmake will now create a symlink for the compile_commands.json file to a location where it will be discoverable by LSPs. (@LilyIsTrans)
```

Comment:
```suggestion
    * CMake will now create a symlink for the compile_commands.json file to a location where it will be discoverable by LSPs. (@LilyIsTrans)
```